Crash trying to run application after installing JsoupFeature

Hey! thanks for building this feature, it's just what I need on my project. Unfortunately when I try to run the application it crashes on startup. I tried with an empty project (just an HttpClient with just this feature installed) but the problem persists.

Here is the trace:
```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at kotlin.reflect.jvm.internal.calls.CallerImpl$Method.callMethod(CallerImpl.kt:97)
	at kotlin.reflect.jvm.internal.calls.CallerImpl$Method$Static.call(CallerImpl.kt:106)
	at kotlin.reflect.jvm.internal.KCallableImpl.callDefaultMethod$kotlin_reflection(KCallableImpl.kt:166)
	at kotlin.reflect.jvm.internal.KCallableImpl.callBy(KCallableImpl.kt:110)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.callFunctionWithInjection(ApplicationEngineEnvironmentReloading.kt:384)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.executeModuleFunction(ApplicationEngineEnvironmentReloading.kt:330)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.access$executeModuleFunction(ApplicationEngineEnvironmentReloading.kt:33)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading$instantiateAndConfigureApplication$1$$special$$inlined$forEach$lambda$1.invoke(ApplicationEngineEnvironmentReloading.kt:275)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading$instantiateAndConfigureApplication$1$$special$$inlined$forEach$lambda$1.invoke(ApplicationEngineEnvironmentReloading.kt:33)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.avoidingDoubleStartupFor(ApplicationEngineEnvironmentReloading.kt:310)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.access$avoidingDoubleStartupFor(ApplicationEngineEnvironmentReloading.kt:33)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading$instantiateAndConfigureApplication$1.invoke(ApplicationEngineEnvironmentReloading.kt:274)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading$instantiateAndConfigureApplication$1.invoke(ApplicationEngineEnvironmentReloading.kt:33)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.avoidingDoubleStartup(ApplicationEngineEnvironmentReloading.kt:290)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.instantiateAndConfigureApplication(ApplicationEngineEnvironmentReloading.kt:272)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.createApplication(ApplicationEngineEnvironmentReloading.kt:125)
	at io.ktor.server.engine.ApplicationEngineEnvironmentReloading.start(ApplicationEngineEnvironmentReloading.kt:245)
	at io.ktor.server.netty.NettyApplicationEngine.start(NettyApplicationEngine.kt:126)
	at io.ktor.server.netty.EngineMain.main(EngineMain.kt:26)
	at com.asdfasdf.ApplicationKt.main(Application.kt:7)
Caused by: java.lang.NoSuchMethodError: io.ktor.client.HttpClient.getResponsePipeline()Lio/ktor/client/response/HttpResponsePipeline;
	at com.tfowl.ktor.client.features.JsoupFeature$Feature.install(JsoupFeature.kt:57)
	at com.tfowl.ktor.client.features.JsoupFeature$Feature.install(JsoupFeature.kt:50)
	at io.ktor.client.HttpClientConfig$install$3.invoke(HttpClientConfig.kt:72)
	at io.ktor.client.HttpClientConfig$install$3.invoke(HttpClientConfig.kt:16)
	at io.ktor.client.HttpClientConfig.install(HttpClientConfig.kt:90)
	at io.ktor.client.HttpClient.<init>(HttpClient.kt:146)
	at io.ktor.client.HttpClientKt.HttpClient(HttpClient.kt:40)
	at io.ktor.client.HttpClientJvmKt.HttpClient(HttpClientJvm.kt:21)
	at com.asdfasdf.ApplicationKt.module(Application.kt:11)
	at com.asdfasdf.ApplicationKt.module$default(Application.kt:10)
	... 24 more

Process finished with exit code 1
```
I'm using version 1.1.0 and Ktor 1.3.2 It seems to work on Ktor 1.2.5 but breaks on +1.3.0, if you need any more info just tell me.

--- a/README.md
+++ b/README.md
@@ -46,7 +46,7 @@ repositories {
         url "https://dl.bintray.com/t-fowl/ktor/"
     }
 }
-implementation 'com.tfowl.ktor:ktor-jsoup:1.0.0'
+implementation 'com.tfowl.ktor:ktor-jsoup:1.2.0'
 ```
 
 Kotlin DSL
@@ -54,7 +54,7 @@ Kotlin DSL
 repositories {
     maven(url = "https://dl.bintray.com/t-fowl/ktor/")
 }
-implementation("com.tfowl.ktor:ktor-jsoup:1.0.0")
+implementation("com.tfowl.ktor:ktor-jsoup:1.2.0")
 ```
 
 ## License
@@ -62,7 +62,7 @@ implementation("com.tfowl.ktor:ktor-jsoup:1.0.0")
 ```
 MIT License
 
-Copyright (c) 2019 Thomas Fowler
+Copyright (c) 2020 Thomas Fowler
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal

--- a/buildSrc/src/main/kotlin/Dependencies.kt
+++ b/buildSrc/src/main/kotlin/Dependencies.kt
@@ -1,5 +1,5 @@
 object Versions {
-    const val kotlin = "1.3.61"
+    const val kotlin = "1.3.70"
     const val dokka = "0.9.18"
     const val bintray = "1.8.4"
     const val kosogor = "1.0.4"
@@ -9,7 +9,7 @@ object Versions {
 object Libraries {
     const val junit = "junit:junit:4.12"
     const val mockk = "io.mockk:mockk:1.9.3"
-    const val jsoup = "org.jsoup:jsoup:1.12.1"
+    const val jsoup = "org.jsoup:jsoup:1.13.1"
 
     val ktor = Ktor
     val kotlin = Kotlin
@@ -19,7 +19,7 @@ object Libraries {
     }
 
     object Ktor {
-        const val version = "1.2.5"
+        const val version = "1.3.2"
         private const val prefix = "io.ktor:ktor"
 
         const val client = "$prefix-client:$version"

--- a/gradle.properties
+++ b/gradle.properties
@@ -2,4 +2,4 @@ org.gradle.parallel=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8
 
 group=com.tfowl.ktor
-version=1.1.0
+version=1.2.0

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/src/main/kotlin/com/tfowl/ktor/client/features/JsoupFeature.kt
+++ b/src/main/kotlin/com/tfowl/ktor/client/features/JsoupFeature.kt
@@ -3,13 +3,13 @@ package com.tfowl.ktor.client.features
 import io.ktor.client.HttpClient
 import io.ktor.client.HttpClientConfig
 import io.ktor.client.features.HttpClientFeature
-import io.ktor.client.response.HttpResponseContainer
-import io.ktor.client.response.HttpResponsePipeline
+import io.ktor.client.statement.HttpResponseContainer
+import io.ktor.client.statement.HttpResponsePipeline
 import io.ktor.http.ContentType
 import io.ktor.http.contentType
 import io.ktor.util.AttributeKey
-import kotlinx.coroutines.io.ByteReadChannel
-import kotlinx.coroutines.io.readRemaining
+import io.ktor.utils.io.ByteReadChannel
+import io.ktor.utils.io.readRemaining
 import org.jsoup.nodes.Document
 import org.jsoup.parser.Parser
 
@@ -67,7 +67,8 @@ class JsoupFeature internal constructor(val parsers: Map<ContentType, Parser>) {
 
                 val parser = feature.parsers.getValue(matchingType)
 
-                proceedWith(HttpResponseContainer(info, parser.parseInput(body.readRemaining().readText(), "${context.request.url}")))
+                val parsedBody = parser.parseInput(body.readRemaining().readText(), "${context.request.url}")
+                proceedWith(HttpResponseContainer(info, parsedBody))
             }
         }
     }

--- a/src/test/kotlin/com/tfowl/ktor/client/features/JsoupFeatureTests.kt
+++ b/src/test/kotlin/com/tfowl/ktor/client/features/JsoupFeatureTests.kt
@@ -6,10 +6,9 @@ import io.ktor.client.engine.mock.MockEngine
 import io.ktor.client.engine.mock.respond
 import io.ktor.client.request.get
 import io.ktor.http.*
-import io.ktor.util.InternalAPI
-import kotlinx.coroutines.io.jvm.javaio.toByteReadChannel
+import io.ktor.util.KtorExperimentalAPI
+import io.ktor.util.cio.toByteReadChannel
 import kotlinx.coroutines.runBlocking
-import kotlinx.io.core.ExperimentalIoApi
 import org.jsoup.nodes.Document
 import org.jsoup.parser.Parser
 import org.junit.Assert.assertEquals
@@ -18,8 +17,7 @@ import org.junit.Test
 import java.io.InputStream
 
 
-@ExperimentalIoApi
-@InternalAPI
+@KtorExperimentalAPI
 class JsoupFeatureTests {
 
     private data class Resource(val file: String, val contentType: ContentType)

